src/index.js

```diff
@@ -0,0 +1,32 @@
+import './promisify.js'
+
+import process from 'process'
+
+import parseArgs from './parseArgs.js'
+import add from './subcommands/add.js'
+import install from './subcommands/install.js'
+import hook from './subcommands/hook.js'
```

Comment:
maybe add a `./subcommands/index.js` containing:
```javascript
export { default as add } from './add';
export { default as install } from './install';
export { default as hook } from './hook';
```
and then we can import it with:
```javascript
import * as subcommands from './subommands'
```